packages/buidler-core/src/internal/buidler-evm/stack-traces/solidity-errors.ts

```diff
@@ -237,7 +239,7 @@ function getMessageFromLastStackTraceEntry(
   }
 }
 
-export class SolidityError extends Error {
+export class SolidityError extends TransactionExecutionError {
```

Comment:
Makes sense. I thought you were wrapping them, and that would mean losing the stack traces.